test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/MapTest.scala

```diff
@@ -16,18 +16,27 @@ import java.{util => ju}
 
 import org.junit.Test
 import org.junit.Assert._
+import org.junit.Assume._
 
 import org.scalajs.testsuite.javalib.util.concurrent.ConcurrentMapFactory
 import org.scalajs.testsuite.utils.AssertThrows._
+import org.scalajs.testsuite.utils.Platform._
 
 import scala.reflect.ClassTag
 
+final case class TestObj(num: Int)
+
 trait MapTest {
-  import MapTest._
 
   def factory: MapFactory
 
+  def testKey(i: Int): TestObj = TestObj(i)
+
+  def SIE[K, V](key: K, value: V): ju.Map.Entry[K, V] =
```

Comment:
I think `testKey` needs to be in `trait MapTest` or `trait MapFactory` so it can be overridden to use in `IdentityHashMapTest`. The second requires import of `MapTest` so I'm not sure which is best, neither seems Ideal. `MapFactory` is where overriding is occurring so maybe that makes the most sense.

Reply:
Moving to the factory didn't seem to work so leaving in `trait MapTest` for now.